src/styles/download.scss

```diff
@@ -27,3 +27,7 @@
     text-align: justify;
   }
 }
+
+.react-tabs__tab-list {
```

Comment:
move `overflow-x: auto` to https://github.com/nodejs/nodejs.dev/blob/master/src/components/UpcomingReleases/UpcomingReleases.scss#L6-L12